setup.py

```diff
@@ -31,6 +31,8 @@
         "pathlib2>=2.2.1,<3",
         "PyYAML",
         "raven",
+        "tqdm",
```

Comment:
what's the reason for choosing tqdm over clint that we are already using for upload progress?

Reply:
if you think tqdm is better, then we should replace clint with it to make the UI consistent.

Reply:
> what's the reason for choosing tqdm over clint that we are already using for upload progress?

No particular reason: I forgot that we used `clint` 😛But I've noticed that sometimes the progress bar during `floyd data  upload` get stuck, I can use this opportunity to investigate if I'll get the same experience with `tqdm`.